tests/cypress/src/DashTable.ts

```diff
@@ -3,10 +3,20 @@ export default class DashTable {
         return cy.get(`#table tbody tr td.column-${column}`).eq(row);
     }
 
+    static focusCell(row: number, column: number) {
+        // somehow we need to scrollIntoView AFTER click, or it doesn't
+        // work right. Why?
+        return this.getCell(row, column).click().scrollIntoView();
```

Comment:
You'd think we should first scroll the cell into view, and THEN click it... but that doesn't solve the problem (that on my machine we can't type into the cell because Cypress thinks it's hidden), it's only solved by scrolling after the click. I gave up trying to figure out why...

Reply:
I find this weird. Never had to scroll into view before. Normally Cypress scrolls into view automatically -- it's disabled for certain tests by default (e.g. with virtualization) as it caused issues.